xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererMediaCodecSurface.cpp

```diff
@@ -212,13 +217,14 @@ void CRendererMediaCodecSurface::ReorderDrawPoints()
     dstRect.x1, dstRect.y1, dstRect.Width(), dstRect.Height(),
     adjRect.x1, adjRect.y1, adjRect.Width(), adjRect.Height());
 
+  // Assumption: SurfaceFlinger starts PTS processing on arrival of first frame.
+  // To have the best match between kodi / android clock we start on first RenderUpdate call.
+  // Beside this the first frame should not be rendered before the SurfaceView is resized.
```

Comment:
Just to clarify: FlipPage is RenderManger, RenderUpdate is GUI loop.
RenderManager / FlipPage is for MediaCodec Surface the "correct" place because video is independent rendered from GUI.

I'll remove the ReleaseBuffer in RenderUpdate as its not really visible what I tried to archieve.